cookbooks/bcpc-hadoop/attributes/hdfs.rb

```diff
@@ -118,28 +118,62 @@
     '30000'
 end
 
-# Default HDFS Directory Quotas
-default['bcpc']['hadoop']['hdfs'].tap do |hdfs|
-  hdfs['user']['space_quota'] = '30G'
-  hdfs['groups']['space_quota'] = '50G'
+# Default HDFS User Directory Configuration
+default['bcpc']['hadoop']['hdfs']['user'].tap do |user|
+  user['home'] = '/user'
+  user['group'] = 'supergroup'
+  user['perms'] = '0700'
+  user['space_quota'] = '30G'
+  user['ns_quota'] = 'NO_QUOTA'
+  user['dirinfo'] = {
+    ## Example dir_info entry:
+    # user_dirname: {
+    #   owner: 'user',
```

Comment:
Clarify `owner` will always be set to the user we pass and remove this example from the comment.